modules/lib/maintainers.nix

```diff
@@ -223,6 +223,16 @@
     github = "nurelin";
     githubId = 5276274;
   };
+  oxalica = {
+    email = "oxalicc@pm.me";
+    github = "oxalica";
+    githubId = 14816024;
+    name = "oxalica";
+    keys = [{
+      longkeyid = "ed25519/0x7571654CF88E31C2";
+      fingerprint = "F90F FD6D 585C 2BA1 F13D  E8A9 7571 654C F88E 31C2";
+    }];
+  };
```

Comment:
Shouldn't be needed as you're already in nixpkgs.

Reply:
Fixed.